### PR TITLE
thingy53: Add cpuappns to platform_exclude

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds thingy53_nrf5340_cpuappns to platform_exclude for samples and applications that provide support for Thingy:53. The non-secure application core is not yet fully supported.
Change also introduces missing ZDebugRTT configuration to sample.yaml in machine_learning application.

Jira: NCSDK-11377

### CODE REVIEW

File: applications/machine_learning/sample.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ tests:
   applications.machine_learning.zdebug:
     build_only: true
     platform_allow: nrf52840dk_nrf52840 thingy52_nrf52832 thingy53_nrf5340_cpuapp
+    platform_exclude: thingy53_nrf5340_cpuappns
```
Comment (Damian-Nordic) [inline applications/machine_learning/sample.yaml:8]: Hasn't "ns" suffix been replaced with "_ns"? The scripts seem to support both, but all docs now refer to "_ns" versions.

Comment (MarekPieta) [reply applications/machine_learning/sample.yaml:8]: It seems like we cannot execute CMake for `thingy53_nrf5340_cpuapp_ns` (it states that board is not supported). Maybe Thingy:53 still refers to old naming convention, but in that case we must use it too.

**Review by shantha-14 — CHANGES_REQUESTED** [review]
Have to double check if platform_exclude is actually used. Also sample owners have to agree to this change and re-do testing

File: applications/machine_learning/sample.yaml (hunk 1 above)
Comment (greg-fer) [reply applications/machine_learning/sample.yaml:8]: This was changed by @mbolivar-nordic  if memory serves me right.

**Comment by greg-fer** [comment]
Please add entry to changelog or remove the related label if the entry is not needed. Thanks.

**Review by shantha-14 — APPROVED** [review]
Since this is exclude list update, I will give for +1 for PR, for future a discussion will be held and agreed.

File: applications/machine_learning/sample.yaml (hunk 1 above)
Comment (tejlmand) [reply applications/machine_learning/sample.yaml:8]: Correct, boards has been renamed from `ns` --> `_ns`, for the  nRF5340dk in this commit:
https://github.com/nrfconnect/sdk-zephyr/commit/b8c9dc169e63dd37536d7381103a038fa79a69f2

But apparently Thingy53 was not part of that renaming, it still has the old `ns` naming scheme.

**Review by pdunaj — CHANGES_REQUESTED** [review]
fix _ns boards postfix

**Comment by tejlmand** [comment]
> fix _ns boards postfix

Thingy53 still uses old naming scheme :disappointed: 
https://github.com/nrfconnect/sdk-zephyr/tree/master/boards/arm/thingy53_nrf5340

**Review by tejlmand — APPROVED** [review]

**Comment by MarekPieta** [comment]
@pdunaj I think we must leave it as it is for now, since Thingy:53 uses old naming scheme (added a ticket to update it at some point)

**Review by KAGA164 — APPROVED** [review]

**Comment by pdunaj** [comment]
> @pdunaj I think we must leave it as it is for now, since Thingy:53 uses old naming scheme (added a ticket to update it at some point)

Indeed it does. So far for having things magically updated when global change is applied.

**Review by pdunaj — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

**Pull request merged**